cargo-shuttle/src/init.rs

```diff
@@ -155,6 +155,65 @@ impl ShuttleInit for ShuttleInitTide {
     }
 }
 
+pub struct ShuttleInitPoem;
+
+impl ShuttleInit for ShuttleInitPoem {
+    fn set_cargo_dependencies(
+        &self,
+        dependencies: &mut Table,
+        manifest_path: &Path,
+        url: &Url,
+        get_dependency_version_fn: GetDependencyVersionFn,
+    ) {
+        set_inline_table_dependency_features(
+            "shuttle-service",
+            dependencies,
+            vec!["web-poem".to_string()],
+        );
+
+        set_key_value_dependency_version(
+            "poem",
+            dependencies,
+            manifest_path,
+            url,
+            get_dependency_version_fn,
+        );
+
+        set_inline_table_dependency_version(
+            "tracing-subscriber",
+            dependencies,
+            manifest_path,
+            url,
+            get_dependency_version_fn,
+        );
+
+        set_inline_table_dependency_features(
+            "tracing-subscriber",
+            dependencies,
+            vec!["env-filter".to_string()],
+        );
+    }
+
+    fn get_boilerplate_code_for_framework(&self) -> &'static str {
+        indoc! {r#"
+        use poem::{get, handler, middleware::Tracing, EndpointExt, Route};
+
+        #[handler]
+        fn hello_world() -> &'static str {
+            "Hello, world!"
+        }
+
+        #[shuttle_service::main]
+        async fn poem() -> shuttle_service::ShuttlePoem<impl poem::Endpoint> {
+            let _ = tracing_subscriber::fmt().try_init();
```

Comment:
This is a great addition! We should probably bake this into shuttle though, which means a separate project (#280) :sweat_smile: Will this PR be complete if we remove this line?

Reply:
Thanks for the feedback! That makes sense. Should I just roll back the tracing commit? It'll be ready to go then.

Reply:
That would be great, thanks